cloud/src/autogluon/cloud/utils/ag_sagemaker.py

```diff
@@ -81,8 +81,18 @@ def __init__(self, *args, **kwargs):
         )
 
 
+class AutoGluonImageRealtimePredictor(Predictor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            serializer=NumpySerializer(),
+            deserializer=PandasDeserializer(),
+            **kwargs
+        )
+
+
 # Predictor documentation: https://sagemaker.readthedocs.io/en/stable/api/inference/predictors.html
-# SageMaker can only take in csv format for batch transformation because files need to be easily splitable to  be batch processed.
+# SageMaker can only take in csv format for batch transformation because files need to be easily splitable to be batch processed.
```

Comment:
I saw the default input and output format is x-parquet, but here only takes in csv format. Are they different use scenarios? 

Also, is x-parquet easy to check, like you can open it in sublime or browser, or you need some app to read it? Thanks.

Reply:
So for the image prediction, the batch inference input format is actually x-image. When you are referring to the default input and output format as x-parquet, I believe that's realtime-inference and training.

I don't think you can open an editor and view x-parquet directly.